packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart

```diff
@@ -479,6 +479,8 @@ Future<void> main() async {
   // we don't have an embedder yet this is the lowest-most layer we can put
   // this stuff in.
   await ui.webOnlyInitializeEngine();
+  (ui.window as dynamic).debugOverrideDevicePixelRatio(3.0);
```

Comment:
This correctly sizes the window to match the flutter tester default

Reply:
We should probably leave a `TODO` to remove the need for `dynamic`.

/cc @hterkelsen